velox/dwio/parquet/tests/duckdb_reader/ParquetReaderTest.cpp

```diff
@@ -199,14 +193,14 @@ TEST_F(ParquetReaderTest, dateRead) {
   auto type = reader->typeWithId();
   EXPECT_EQ(type->size(), 1ULL);
   auto col0 = type->childAt(0);
-  EXPECT_EQ(col0->type->kind(), TypeKind::DATE);
+  EXPECT_EQ(col0->type->kind(), TypeKind::INTEGER);
```

Comment:
You can check the type->name() here for date I think to check for DATE type in Parquet while reading.